src/snark/Makefile

```diff
@@ -19,7 +19,7 @@ DEPINST = depinst
 
 CXXFLAGS += -I$(DEPINST)/include -Isrc
 LDFLAGS += -L$(DEPINST)/lib -Wl,-rpath,$(DEPINST)/lib
-LDLIBS += -lgmpxx -lgmp -lboost_program_options
+LDLIBS += -lgmpxx -lgmp -lboost_program_options-mt -lsodium
```

Comment:
what's the difference between `-lboost_program_options` and `-lboost_program_options-mt`?

Reply:
The `-mt` tag indicates the library was [built with multithreading](http://www.boost.org/doc/libs/1_62_0/more/getting_started/unix-variants.html#library-naming), which ours is.